src/sentry/options/defaults.py

```diff
@@ -435,6 +435,14 @@
 register("github-app.client-id", flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE)
 register("github-app.client-secret", flags=FLAG_CREDENTIAL | FLAG_PRIORITIZE_DISK)
 
+# Github Enterprise Integration
+register(
+    "github-enterprise-app.alert-rule-action",
+    type=Bool,
+    default=False,
+    flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
```

Comment:
qq - what's the reason you chose to add `FLAG_PRIORITIZE_DISK`?

Here is a note Filippo had for the option I created:

https://sentry.slack.com/archives/C039ZGT5K/p1711387859943059?thread_ts=1711145398.370849&cid=C039ZGT5K

Reply:
I'll remove it and also add `FLAG_MODIFIABLE_BOOL`. I think they're really trying to get people to use the options automator instead of changing options through _admin

Reply:
Yeah feel like the options documentation could use a refresh